js/wp-seo-term-scraper.js

```diff
@@ -0,0 +1,318 @@
+/* global YoastSEO: true, wp, wpseoTermScraperL10n, ajaxurl, tinyMCE */
```

Comment:
This is almost the same as the post scraper, is there no way to abstract the common functionality?